python_docs_theme/__init__.py

```diff
@@ -20,6 +21,11 @@ def _asset_hash(path: str) -> str:
 
 
 def _add_asset_hashes(static: List[str], add_digest_to: List[str]) -> None:
+    if sphinx.version_info >= (7, 1):
```

Comment:
There's already a check on line 41, could we modify that to read:

```
    (4,) >= sphinx.version_info > (7, 1)
```

Reply:
(FWIW, that conditional is always going to be False)

Reply:
Sorry, I'm currently on a train--should've waited until I got back to my computer to make sure I got the check right, sorry!

Reply:
Don't be! The review is appreciated! :)